tests/integration/monaibundle/test_infer.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import os
+import unittest
+
+import requests
+import torch
+
+# from tests1.integration import SERVER_URI
+
+
+SERVER_URI = f"http://0.0.0.0:{os.getenv('MONAILABEL_SERVER_PORT', '8000')}"
```

Comment:
may be not needed.. you can use `from tests.integration import SERVER_URI`

Reply:
Yes, modified. Thanks